qlib/workflow/recorder.py

```diff
@@ -308,9 +311,12 @@ def save_objects(self, local_path=None, artifact_path=None, **kwargs):
 
     def load_object(self, name):
         assert self.uri is not None, "Please start the experiment and recorder first before using recorder directly."
-        path = self.client.download_artifacts(self.id, name)
-        with Path(path).open("rb") as f:
-            return pickle.load(f)
+        try:
+            path = self.client.download_artifacts(self.id, name)

```

Comment:
Are you sure the `download_artifacts` will only raise  OSError ?

Reply:
LocalArtifactRepository will raise OSError, but I am not sure about others.